### DISCUSSION TITLE
User not found in live app

### DISCUSSION THREAD

**Discussion by applicationuser** [Budibase/budibase#10455] [Help] [answered]
Hello,

Budibase Cloud, App version: 2.5.7

New user added to app not visible in app user table. 

User can login to budibase and access app but no record of the user in app user table.  User cannot use app as app data access is controlled through additional properties in user table. 

same issue in multiple apps.  users are part of sso.

User can be found in dev app user table and app can be published to resolve the issue for now,  

Thanks,

**Comment by ConorWebb96** [comment]
Hey @applicationuser,

The issue you are describing should be resolved sometime this week whenever 2.5.9 is released. Currently the user's table isn't reactive and an upgrade to resolve these issues from occurring has been implemented.

At the moment you will need to publish anytime data within the users table is changed.

For reference here is the issue created around this: https://github.com/Budibase/budibase/issues/10216

I hope this helps.

**Reply by applicationuser** [reply]
Hello @ConorWebb96 

Thank you for the reply, Good to know fix is on the way.   Its been tough to manage publishing every time new user needs to be needed while the app is actively used in live environment with data adds/updates.  

I see that 2.5.9 is released, and when tested on app using the latest version , still has the issue 
![image](https://user-images.githubusercontent.com/42922833/235666809-db079a39-7cc7-44ee-9cec-9e9f6b73283e.png)

**Reply by ConorWebb96** [reply, marked as answer]
Hey @applicationuser,

Apologies, I made a mistake. That is the most recent release mentioned: 2.5.9. This fix should be available on the next minor incremented release being 2.6 +. I think this is likely to be released by the end of this week, possibly early next week. If you are still experiencing issues after that release let us know.

I hope this helps

**Reply by applicationuser** [reply]
@ConorWebb96 

tested on latest app version 2.6.6 , still has issue with user access to live app. 

any idea when the fix will be released?
